<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>966</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget_principal">
   <widget class="QTabWidget" name="Onglet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>560</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="vol_immersif">
     <attribute name="title">
      <string>Vol immersif</string>
     </attribute>
     <widget class="QPushButton" name="VideoStreaming">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>START/STOP VIDEO STREAMING</string>
      </property>
     </widget>
     <widget class="QPushButton" name="emergency_stop">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>480</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>EMERGENCY STOP</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>681</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="cameralayout"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>200</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="maplayout"/>
     </widget>
     <widget class="QPushButton" name="Photo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>480</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TAKE PICTURE</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="batterylayout"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="batterylayout_2"/>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BASE</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DRONE</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>20</width>
        <height>471</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>20</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout4"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>20</y>
        <width>20</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout3"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout1"/>
     </widget>
     <widget class="QLabel" name="Name_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SIGNAL RF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vol_vue">
     <attribute name="title">
      <string>Vol Ã  vue</string>
     </attribute>
     <widget class="QPushButton" name="emergency_stop_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>480</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>EMERGENCY STOP</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>681</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="horizon_layout"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>200</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="maplayout2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="batterylayout2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="batterylayout2_2"/>
     </widget>
     <widget class="QLabel" name="Name2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BASE</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name2_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DRONE</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>20</width>
        <height>471</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>20</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout4_2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>20</y>
        <width>20</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout3_2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_15">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout2_2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_16">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="powerlayout1_2"/>
     </widget>
     <widget class="QLabel" name="Name_4">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SIGNAL RF</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
